src/layouts/MainLayout.tsx

```diff
@@ -23,8 +23,8 @@ export const MainLayout = () => {
   return (
     <CookiesProvider>
       <ZonesAndFspMapContextProvider>
-        <FileBrowserContextProvider fspName={fspName} filePath={filePath}>
-          <OpenFavoritesProvider>
+        <OpenFavoritesProvider>
```

Comment:
Did AI swap the order of these providers, or was there some underlying reason why they needed to be nested in the reverse order?